infra/shared/data-scripts/user-data-server.sh

```diff
@@ -86,3 +86,91 @@ consul kv put -token-file=$CONSUL_BOOTSTRAP_TOKEN nomad_user_token "$(cat $NOMAD
 
 echo "ACL bootstrap end"
 
+sudo systemctl restart vault
+
+sleep 10
+
+export VAULT_ADDR=http://127.0.0.1:8200
+echo "export VAULT_ADDR=http://127.0.0.1:8200" >> ~/.bashrc
+
+touch /etc/vault.d/init.file
+vault operator init > /etc/vault.d/init.file
+
+echo -e '\e[38;5;198m'"++++ Auto unseal vault"
+
+for i in $(cat /etc/vault.d/init.file | grep Unseal | cut -d " " -f4 | head -n 3); do vault operator unseal $i; done
+VAULT_TOKEN=$(grep 'Initial Root Token' /etc/vault.d/init.file | cut -d ':' -f2 | tr -d ' ')
```

Comment:
To be more consistent with rest of the code rename the var to `VAULT_BOOTSTRAP_TOKEN` or `VAULT_ROOT_TOKEN`